### DISCUSSION TITLE
read values from stdin lines

### DISCUSSION THREAD

**Discussion by eadmaster** [Qalculate/libqalculate#486] [Q&A]
is it possible to read values from stdin lines and pass them to a function?
e.g.
````
$ cat a
1
2
3
$ cat a | qalc "mean(load(stdin))"
2  # expected output
````

**Comment by howaboutuser** [comment]
cat a|qalc "mean(load(/dev/stdin))"
qalc "mean(load(/dev/stdin))"<a
